src/vivarium_gates_nutrition_optimization_child/components/wasting.py

```diff
@@ -121,18 +130,6 @@ def setup(self, builder: Builder):
         self.scenario = scenarios.INTERVENTION_SCENARIOS[
             builder.configuration.intervention.child_scenario
         ]
-        self._register_on_time_step_prepare_listener(builder)
-
-    def _get_population_view(self, builder: Builder) -> PopulationView:
-        return builder.population.get_view(
-            [self.propensity_column_name, self.previous_wasting_column, self.wasting_column]
-        )
-
-    def _register_on_time_step_prepare_listener(self, builder: Builder) -> None:
-        # we want to reset propensities before updating previous state column
-        builder.event.register_listener(
-            "time_step__prepare", self.on_time_step_prepare, priority=4
```

Comment:
You haven't defined the time-step prepare priority property.

Reply:
It's the first property in the WastingTreatment component.